src/buildVariables.js

```diff
@@ -130,18 +133,60 @@ const buildUpdateVariables = (resource, aorFetchType, params, queryType) =>
       return acc;
     }
 
+    const type = introspectionResults.types.find(
```

Comment:
Can we abstract lines 136 to 155 into a function so that it can be reused in `buildCreateVariables` from lines 171 to 189?